src/engraving/dom/measurerepeat.cpp

```diff
@@ -193,6 +193,15 @@ String MeasureRepeat::accessibleInfo() const
     return muse::mtrc("engraving", "%1; Duration: %n measure(s)", nullptr, numMeasures()).arg(EngravingItem::accessibleInfo());
 }
 
+//---------------------------------------------------------
+//   subtypeUserName
+//---------------------------------------------------------
+
+muse::TranslatableString MeasureRepeat::subtypeUserName() const
+{
+    return muse::TranslatableString("engraving", "%1 measure(s)").arg(numMeasures());
```

Comment:
Even better:
```suggestion
    return muse::TranslatableString("engraving", "%n measure(s)", nullptr, numMeasures());
```
This way, translators can provide different translations for singular/plural/… based on the value of `numMeasures()`, for example "1 measure" and "2 measures".

Reply:
Oops, I was apparently wrong here! TranslatableString apparently has no fourth argument at all. If you don't mind, I'll push a commit to fix that :)

Reply:
I don't mind either way, I was able to call the `numMeasures()` argument over `translatedSubtypeUserName` instead.